src/NLog.Targets.Seq/Layouts/TraceIdLayout.cs

```diff
@@ -0,0 +1,19 @@
+using System.Diagnostics;
+using System.Text;
+using NLog.Layouts;
+
+namespace NLog.Targets.Seq.Layouts
+{
+    class TraceIdLayout: Layout
```

Comment:
Seems rather similar to `SpanIdLayout`

Reply:
Yeah, could definitely factor this out; I'll take a quick look tomorrow :)